<ui version="4.0" >
 <class>phosGui</class>
 <widget class="QMainWindow" name="phosGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="tab_modules" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1421</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_module_0" >
     <attribute name="title" >
      <string>Module 0</string>
     </attribute>
     <widget class="QTextBrowser" name="log_viewer" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>490</y>
        <width>1021</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame_rcu0" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>40</y>
        <width>641</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="fee_1_b" >
       <property name="geometry" >
        <rect>
         <x>240</x>
         <y>0</y>
         <width>16</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text" >
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rcu_update_status" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>500</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Update status</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fee_1_a" >
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>0</y>
         <width>16</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text" >
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rcu_toggle" >
       <property name="geometry" >
        <rect>
         <x>500</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Toggle On/Off</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rcu_view" >
       <property name="geometry" >
        <rect>
         <x>500</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>RCU View</string>
       </property>
      </widget>
      <widget class="Line" name="rcu_line_3" >
       <property name="geometry" >
        <rect>
         <x>470</x>
         <y>10</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="rcu_line_2" >
       <property name="geometry" >
        <rect>
         <x>400</x>
         <y>10</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_41" >
       <property name="geometry" >
        <rect>
         <x>420</x>
         <y>0</y>
         <width>16</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text" >
        <string>A</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_42" >
       <property name="geometry" >
        <rect>
         <x>450</x>
         <y>0</y>
         <width>16</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text" >
        <string>B</string>
       </property>
      </widget>
      <widget class="Line" name="rcu_line_1" >
       <property name="geometry" >
        <rect>
         <x>220</x>
         <y>10</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="rcu_line_0" >
       <property name="geometry" >
        <rect>
         <x>40</x>
         <y>10</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_rcu0" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>10</y>
         <width>16</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text" >
        <string>R C U 0</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5" >
      <property name="geometry" >
       <rect>
        <x>1280</x>
        <y>20</y>
        <width>101</width>
        <height>691</height>
       </rect>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6" >
      <property name="geometry" >
       <rect>
        <x>1150</x>
        <y>20</y>
        <width>101</width>
        <height>691</height>
       </rect>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_rcu_top" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>0</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="Line" name="title_line_3" >
       <property name="geometry" >
        <rect>
         <x>470</x>
         <y>0</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="title_line_2" >
       <property name="geometry" >
        <rect>
         <x>400</x>
         <y>0</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="title_line_1" >
       <property name="geometry" >
        <rect>
         <x>220</x>
         <y>0</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="title_line_0" >
       <property name="geometry" >
        <rect>
         <x>40</x>
         <y>0</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_fee_a" >
       <property name="geometry" >
        <rect>
         <x>61</x>
         <y>10</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text" >
        <string>Front End Cards Branch A</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_fee_b" >
       <property name="geometry" >
        <rect>
         <x>240</x>
         <y>10</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text" >
        <string>Front End Cards Branch B</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tru" >
       <property name="geometry" >
        <rect>
         <x>420</x>
         <y>10</y>
         <width>62</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text" >
        <string>TRU A/B</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_actions" >
       <property name="geometry" >
        <rect>
         <x>530</x>
         <y>10</y>
         <width>62</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text" >
        <string>Actions</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
